### PR TITLE
Gh action terrscan_atlantis release

### PR DESCRIPTION
**Pull request by dev-gaur**

### CODE REVIEW

File: scripts/atlantis/docker-build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```
Comment (jlk) [inline scripts/atlantis/docker-build.sh:2]: Maybe take `scripts/docker-build.sh` and tweak it so it takes `DOCKER_REPO` and `DIR` as parameters?

File: scripts/atlantis/docker-push-latest-tag.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
```
Comment (jlk) [inline scripts/atlantis/docker-push-latest-tag.sh:2]: same for this guy, see if `scripts/docker-push*` can be adjusted for several images

**Review by jlk — APPROVED** [review]
I'm not going to request changes as I don't want to block the merge, but you can improve the scripts as I made comments. :)

File: scripts/atlantis/docker-build.sh (hunk 1 above)
Comment (dev-gaur) [reply scripts/atlantis/docker-build.sh:2]: I did have a PR for something like that : https://github.com/accurics/terrascan/pull/555

I'll re-open it and make these changes there later. savvy?

**Pull request merged**